src/components/UserInfo.js

```diff
@@ -1,18 +1,40 @@
+import { api } from "../components/Api.js";
```

Comment:
* From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.